cdap-unit-test/src/test/java/co/cask/cdap/security/AuthorizationTest.java

```diff
@@ -489,6 +491,13 @@ public Boolean call() throws Exception {
         return dummyAppManager.isRunning(serviceId.toId());
       }
     }, 5, TimeUnit.SECONDS);
+    ServiceManager greetingService = dummyAppManager.getServiceManager(serviceId.getProgram());
+    // alice should be able to set instances for the program
+    greetingService.setInstances(2);
+    Assert.assertEquals(2, greetingService.getProvisionedInstances());
+    // alice should also be able to save runtime arguments for all future runs of the program
+    Map<String, String> args = ImmutableMap.of("key", "value");
+    greetingService.setRuntimeArgs(args);
```

Comment:
its not possible to do that today in Testbase. The main point of this test is to ensure that there is no authorization error. I am hesitant to add functionality in TestBase for this, given that we may deprecate the runtime arguments api in future.